core/src/test/java/io/grpc/internal/DelayedClientTransportTest.java

```diff
@@ -96,8 +96,10 @@
 
   @Before public void setUp() {
     MockitoAnnotations.initMocks(this);
-    when(mockRealTransport.newStream(same(method), same(headers))).thenReturn(mockRealStream);
-    when(mockRealTransport2.newStream(same(method2), same(headers2))).thenReturn(mockRealStream2);
+    when(mockRealTransport.newStream(same(method), same(headers), any(CallOptions.class)))
+        .thenReturn(mockRealStream);
```

Comment:
We should probably create a `CallOptions`, change every `newStream()` call to the new one that passes in the `CallOptions`, and verify that it's propagated to the transport, like what we are doing for `method` and `headers`.